builtin/mainmenu/tab_online.lua

```diff
@@ -17,14 +17,31 @@
 
 local function get_sorted_servers()
 	local servers = {
+		lan = {},
 		fav = {},
 		public = {},
 		incompatible = {}
 	}
 
+	local merged_serverlist = table.copy(serverlistmgr.servers)
```

Comment:
this is really clean 👍

Reply:
Really? I would have thought that it would be better to find a way to merge the tables without making a copy and merging it that way. This was just supposed to be temporary until I found a way to do that.

Reply:
it seems fine, I like that the diff for tab_online.lua is small more than anything.